.circleci/config.yml

```diff
@@ -29,6 +29,29 @@ jobs:
       - run: git push origin $CIRCLE_SHA1:refs/heads/master
 
 workflows:
+  # Daily monitoring of general application functionality
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only: develop
+    jobs:
+      - test
+      # Run our Rainforest tests. `run_group_id` should point to the Run Group you
+      # want to use for your releases. `environment_id` should point to your Staging or QA
+      # environment (the one we just deployed to).
```

Comment:
We didn't "just deploy to" any environment if this is running on a schedule 😉

Reply:
right.